hypothesis-python/src/hypothesis/core.py

```diff
@@ -239,7 +239,23 @@ def invalid(message):
         def wrapped_test(*arguments, **kwargs):
             raise InvalidArgument(message)
 
+        def _get_fuzz_target() -> Callable[
+            [Union[bytes, bytearray, memoryview, BinaryIO]], Optional[bytes]
+        ]:
+            def fuzz_one_input(
+                buffer: Union[bytes, bytearray, memoryview, BinaryIO]
+            ) -> Optional[bytes]:
+                return None
+
+            fuzz_one_input.__doc__ = HypothesisHandle.fuzz_one_input.__doc__
+            return fuzz_one_input
+
         wrapped_test.is_hypothesis_test = True
+        wrapped_test.hypothesis = HypothesisHandle(
+            wrapped_test, 
```

Comment:
I think this should be user's real `test` function, rather than the fake `wrapped_test`.

(Usually the difference shouldn't matter, but I can imagine someone doing introspection on `test.hypothesis.inner_test` and being surprised that they don't get the original test back.)

`test` isn't available in `is_invalid_test`, but we can fix that easily enough by changing the `name` parameter to `test`, fixing the call site, and then doing `name = test.__name__` inside.